### ISSUE TITLE
Are Type-Value frames usable in WebTransport

### ISSUE THREAD

**Issue by ianswett** [ietf-wg-webtrans/draft-ietf-webtrans-http3#26]
I believe the QUIC WG decided that TV frames were fine in extensions, but that we weren't going to use them in the core spec?  Others disagreed with my recollection/understanding.

Subsequent discussion from Lucas and Alan indicated that TV frames are not allowed in HTTP/3, but it is allowed to have a 0 length frame which indicated the rest of the stream was WebTransport data.  I'm not sure it matters, since either way a peer which doesn't understand the frame is going to parse the stream.  Those that skip the frame will start parsing another frame after it and the parser will enter an unknown state.  Similarly, the behavior for TV is going to be unexpected/unknown.

_Originally posted by @ianswett in https://github.com/vasilvv/webtransport/issues/24#issuecomment-760704522_

**Comment by afrind** [comment]
I had forgotten that at some point H3 removed the restriction on zero-length DATA frames.  That doesn't prevent an extension from frame using length 0 to indicate the remainder of the stream is unframed however.  I think my implementation would be  easier if we used 0 length, but I've made it work with TV frames and can live with it.

Another argument for having a length here is if this extension were to allow an optional header block or other variable length field at the beginning of a WT stream, there's currently no way to know how long it is.
